src/app/shared/dynamic-form/components/form-datetime/form-datetime.component.html

```diff
@@ -28,7 +28,9 @@
     <mat-datepicker #picker [calendarHeaderComponent]="header"></mat-datepicker>
   </mat-form-field>
   <mat-form-field appearance="outline" floatLabel="always" class="time">
-    <mat-label>{{ 'Select time' | translate }}</mat-label>
+    <mat-label>{{
+      config.timeLabel !== undefined ? config.timeLabel : ('Select time' | translate)
```

Comment:
since it comes from form-data files, `config.timeLabel` is a translation key
so this should be:
(config.timeLabel || 'Select time') | translate

Reply:
Fixed. Now it has text only if it's added in the config.